src/clojuratica/base/parse.clj

```diff
@@ -94,9 +94,9 @@
 
 (defn bound-map [f coll]
   (cond
-    (options/flag? dynamic-vars/*options* :vectors) (vec-bound-map f coll)
-    (options/flag? dynamic-vars/*options* :seqs     (seq-bound-map f coll))
-    (options/flag? dynamic-vars/*options* :seq-fn   (seq-fn-bound-map f coll))))
+    (options/flag? dynamic-vars/*options* :vectors)  (vec-bound-map f coll)
+    (options/flag? dynamic-vars/*options* :seqs)     (seq-bound-map f coll)

```

Comment:
This looked to me like parens in a wrong place

Reply:
Seems likely, good catch...